### PR TITLE
add satysfi.0.0.3

### PR DESCRIPTION
**Pull request by na4zagin3**
This PR adds satysfi.0.0.3.

I’m still wondering descriptions of `maintainer`, `synopsis`, and `description` sections.
- Would it be better to have `maintainer: "@gfngfn"`?
- Are there better texts for `synopsis` and `description`?

I’d be happy if I could get comments and improve the content.

### CODE REVIEW

File: packages/satysfi/satysfi.0.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,83 @@
+opam-version: "2.0"
+name: "satysfi"
+version: "0.0.3"
+maintainer: "SAKAMOTO Noriaki <mrty.ityt.pt@gmail.com>"
```
Comment (nekketsuuu) [inline packages/satysfi/satysfi.0.0.3/opam]: You can write multiple maintainers according to OPAM2 manual :) https://opam.ocaml.org/doc/Manual.html#opam

**Comment by nekketsuuu** [comment]
trying to re-run CI testing

File: packages/satysfi/satysfi.0.0.3/opam [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+synopsis: "SATySFi: New typesetting system with static type system"
+description: """
+[SATySFi] is a typesetting system with a static type system.
+
+SATySFi is distributed under the LGPL-3.0 license.
+
+
+  [SATySFi]: https://github.com/gfngfn/SATySFi"""
+url {
+  src: "https://github.com/gfngfn/SATySFi/archive/v0.0.3.tar.gz"
```
Comment (nekketsuuu) [inline packages/satysfi/satysfi.0.0.3/opam:80]: Would you add checksum here?

**Comment by nekketsuuu** [comment]
@na4zagin3 Would you add this package to `.travis.yml` for testing installation, please?

**Comment by nekketsuuu** [comment]
Memo: gfngfn/satysfi#147 adds synopsis and description to the OPAM file.

**Comment by na4zagin3** [comment]
@nekketsuuu Oh, I’m so sorry for forgetting replying your messages. I’m now rebasing onto the current master, addressing the comments.

File: packages/satysfi/satysfi.0.0.3/opam (hunk 1 above)
Comment (na4zagin3) [reply packages/satysfi/satysfi.0.0.3/opam]: TIL! Added gfngfn.

File: packages/satysfi/satysfi.0.0.3/opam (hunk 2 above)
Comment (na4zagin3) [reply packages/satysfi/satysfi.0.0.3/opam:80]: Sure! Added sha256 and sha512 checksums.

**Comment by na4zagin3** [comment]
Hmm, the container lacks ruby. Let me fix that.

**Comment by na4zagin3** [comment]
~~@nekketsuuu I’m new to Travis. Could you please help me by answering my question?
Most samples and documents in the Internet mention `.travis.yml` but not `.travis-Dockerfile.template`. If I simply updated .travis-Dockerfile.template, would Travis automatically use it?~~

EDIT: Okay, I now understand what’s going on.

**Pull request closed without merging**